docs/install/network_options.md

```diff
@@ -58,3 +58,92 @@ cp rke2-cilium.yml /var/lib/rancher/rke2/server/manifests/
+
+To deploy Multus, use following config:
+
+```
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: rke2-multus
+  namespace: kube-system
+spec:
+  repo: https://rke2-charts.rancher.io
+  chart: rke2-multus
+```
+
+The config needs to be copied over to the manifests directory:
+
+```
+mkdir -p /var/lib/rancher/rke2/server/manifests/
+cp rke2-multus.yml /var/lib/rancher/rke2/server/manifests/
+```
+
+For more information about the options of the Multus config please refer to the [rke2-charts](https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-multus/charts/values.yaml)
+
+
+## Using Multus with the containernetworking plugins
+
+To take advantage of Multus, extra CNI plugins should be used to manage the additional network interfaces attached per pod. That is why, when installing the Multus helm chart, the common CNI plugins maintained by the containernetworking team are deployed (e.g. macvlan CNI). For more information about these plugins, check the [containernetworking plugins](https://www.cni.dev/plugins/current).
+
+As explained in the Multus documentation, to use one of these plugins, a NetworkAttachmentDefinition object would need to be created first pointing to one of these CNI plugins. After that, in the pod annotations, we can refer to it so that Multus can start the process to provide one extra interface to that pod. Please follow this example that configures macvlan cni plugin with multus [example](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/quickstart.md#storing-a-configuration-as-a-custom-resource)
+
+## Using Multus with SRIOV
+
+Using the SRIOV-cni as part of Multus, can help with dataplane acceleration use cases, providing an extra interface in the pod that can achieve very high throughput. SRIOV will not work in all environments and there are several requirements that must be fulfilled to consider the node as sriov capable:
+
+* Physical NIC must support SRIOV (e.g. by checking /sys/class/net/$NIC/device/sriov_totalvfs)
+* The host operating system must activate IOMMU virtualization
+* The vfio driver must be available in the host operating system
+
+SRIOV requires two different Helm charts. One for the crds:
+
+```
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: rke2-sriov-crd
+  namespace: kube-system
+spec:
+  repo: https://rke2-charts.rancher.io
+  chart: rke2-sriov-crd
+```
+
+And one for the sriov operator:
+
+```
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: rke2-sriov
+  namespace: kube-system
```

Comment:
Should we recommend deploying this on it's own namespace?

Reply:
I think it is fine to deploy HelmCharts in kube-system. That does not mean the deployment will end up there too